yt_idefix/api.py

```diff
@@ -5,5 +5,8 @@
     IdefixGrid,
     IdefixVtkDataset,
     IdefixVtkHierarchy,
+    PlutoXdmfHierarchy,
+    PlutoStaticDataset,
```

Comment:
this reminds me; you should add a line to `pyproject.toml` in the `[project.entry-points."yt.frontends"]` section
In general, you want to make sure you've read *all* places where `PlutoVtkDataset` is listed so you can add an entry for `PlutoXdmfDataset` next to it.

Reply:
Done!